sdk/eventhub/event-hubs/package.json

```diff
@@ -47,7 +47,7 @@
     "@types/dotenv": "^6.1.0",
     "@types/long": "^4.0.0",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^8.0.0",
+    "@types/node": "^8.0.37",
```

Comment:
It looks like rush update or rush sync-versions updated more than just amqp-common. Is there a way to revert these changes?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/2208#discussion_r279894726

Reply:
I'll update the onboarding section of the README, since this seems to be a common occurrence. But generally, when onboarding a new library you should make every effort to change your dependency versions to match those of the other projects, rather than updating every other project to match yours.